### PR TITLE
feat: [FFM-4406]: searching against api

### PR DESCRIPTION
**Pull request by crystal-fong**
### Summary
 Sends the search term as part of the query params to search for services against the API rather than in the UI. 

#### Screenshots

https://user-images.githubusercontent.com/95680496/186457384-2239060e-fa4e-42a7-a7cc-dca251ae80b0.mov

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Comment by crystal-fong** [comment]
retrigger titlecheck

File: src/modules/75-cf/components/FlagActivation/__tests__/ServicesList.test.tsx [hunk 1]
```diff
@@ -17,6 +17,7 @@ import mockServiceList from './__data__/mockService'
 import ServicesList from '../ServicesList'
 
 const refetchFlagMock = jest.fn()
+const loadingMessage = 'Loading, please wait...'
```
Comment (laura-mannus) [inline src/modules/75-cf/components/FlagActivation/__tests__/ServicesList.test.tsx:20]: Can we not have this in the strings yaml as well?

**Review by laura-mannus — APPROVED** [review]
Just a question, but LGTM nice work

File: src/modules/75-cf/components/FlagActivation/__tests__/ServicesList.test.tsx (hunk 1 above)
Comment (crystal-fong) [reply src/modules/75-cf/components/FlagActivation/__tests__/ServicesList.test.tsx:20]: I'm not sure where that string comes from, it comes default using `ModalDialog`

Comment (crystal-fong) [reply src/modules/75-cf/components/FlagActivation/__tests__/ServicesList.test.tsx:20]: Just checked, it comes from `common.loading` but the DOM renders `Loading, please wait...` so I can't grab it as `common.loading`

**Review by Kira-Jennings — APPROVED** [review]

**Pull request merged**